install/resources/mybb_theme.xml

```diff
@@ -10568,7 +10568,10 @@ if(use_xmlhttprequest == "1")
 	$("#to").select2({
 		placeholder: "{$lang->search_user}",
 		minimumInputLength: 2,
-		maximumSelectionSize: {$mybb->usergroup['maxpmrecipients']},
+		maximumSelectionSize: function() {
+			max = {$mybb->usergroup['maxpmrecipients']} - $("#bcc").select2('data').length
```

Comment:
Perhaps I'm being blind, but where is `max` defined? There's also a missing semicolon at the end. Should this not be:

```suggestion
			var max = {$mybb->usergroup['maxpmrecipients']} - $("#bcc").select2('data').length;
```

Reply:
It's just defining it for the local scope so doesn't need `var`?

Perhaps you'd prefer this instead of the temporary variable?
```js
return Math.max(1, {$mybb->usergroup['maxpmrecipients']} - $("#bcc").select2('data').length);
```

Reply:
Yeah, that looks cleaner to me. 

I always use `var` (or `let` or `const`) personally.